### PR TITLE
fixed typos, added clarifications

### PR DESCRIPTION
**Pull request by mhaeberli**
-Normalized capitalization of OpenAPS (except where it's a command, file, or directory name) as "OpenAPS" vs openaps or openAps, ...
-Normalized capitalization of "Ethernet"
-Added a bit of mention of mmeowlink as an alternative to CareLink stick for pump access
-Attempted to clarify (and raise questions) related to using nightscout for CGM download, etc

### CODE REVIEW

**Comment by mhaeberli** [comment]
- oh, and fixed a broken link to the nightscout download docs ...

File: docs/docs/walkthrough/phase-0/openaps.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Setting Up openaps and Dependencies
+# Setting Up OpenAPS and Dependencies
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-0/openaps.md]: These ones should be openaps.  They're referring to the https://github.com/openaps/openaps toolkit, which is lowercase.

File: docs/docs/walkthrough/phase-1/log-clean-analyze.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 
 # Phase 1: Logging, Cleaning, and Analyzing Your Data
 
-Phase 1 focuses on accessing, logging, cleaning up, and analyzing data from the pump and CGM. Data fidelity is extremely important, especially when dosing is being considered. Take the time to review what the openaps tools are outputting and carefully compare the logs against your own CareLink and CGM reports.
+Phase 1 focuses on accessing, logging, cleaning up, and analyzing data from the pump and CGM. Data fidelity is extremely important, especially when dosing is being considered. Take the time to review what the OpenAPS tools are outputting and carefully compare the logs against your own CareLink and CGM reports.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/log-clean-analyze.md]: This is also referring to openaps tools and should be lowercase.

**Comment by scottleibrand** [comment]
Lots of instances referring to the openaps toolkit / tools got changed to OpenAPS, which is incorrect.  I noted a few of them, but there are lots more that need changed back.

**Comment by mhaeberli** [comment]
Scott,
Sorry for the extra effort - I'll try to work on this tomorrow.
Are you OK with "Ethernet" vs "ethernet"?
And, as I said in pm to @danamlewis, feel free to reject all if it's
simpler.

**Comment by scottleibrand** [comment]
Yeah, Ethernet is like Internet. It's technically correct to capitalize it, but often not worth correcting. :-)

Most of your edits are good, so just push a fix for the openaps ones and we should be good to merge. 

Thanks!

**Comment by mhaeberli** [comment]
Scott, ok - so (not my own original, but in the pre-existing - instances of
"OpenAps" can also go to "openaps"?)

File: docs/docs/walkthrough/phase-0/hardware.md [hunk 3]
```diff
@@ -128,16 +128,16 @@ Because the Medronic pump collects data directly from the Enlite sensors, OpenAP
...
 
-The RPi has 4 USB ports, an ethernet port, an HDMI port, and a micro USB power-in jack that accepts 2.1 Amp power supplies. In this tutorial, you will need to access the USB ports, micro USB power-in jack, and possibly the Ethernet jack (if wireless failure occurs). You will not require the HDMI port or a monitor.
+The RPi has 4 USB ports, an Ethernet port, an HDMI port, and a micro USB power-in jack that accepts 2.1 Amp power supplies. In this tutorial, you will need to access the USB ports, micro USB power-in jack, and possibly the Ethernet jack (if wireless failure occurs). You will not require the HDMI port or a monitor.
 
 #### Raspberry Pi 2
-Raspberry Pi 2’s have fewer and lower spec components and so draw less power, but require a WiFi adapter to be also purchased. The spec makes no difference to the OpenAps app and so either model is suitable choice.
+Raspberry Pi 2’s have fewer and lower spec components and so draw less power, but require a WiFi adapter to be also purchased. The spec makes no difference to the OpenAPS app and so either model is suitable choice.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-0/hardware.md]: This one is ambiguous. Either OpenAPS or openaps is fine, but definitely not OpenAps. :-)

**Comment by mhaeberli** [comment]
Scott,
Tried to fix OpenAPS vs openAps vs openaps capitalization per your suggestion.
Hope this addresses your concerns...
Thanks!
Martin

**Comment by scottleibrand** [comment]
Perfect, thanks!

**Pull request merged**